core/src/test/scala/unit/kafka/server/KafkaApisTest.scala

```diff
@@ -412,97 +420,640 @@ class KafkaApisTest {
     results.foreach(r => {
       assertEquals(Errors.NONE.code, r.errorCode)
       assertFalse("Should return configs", r.configs.isEmpty)
-      r.configs.forEach(c => {
+      r.configs.asScala.filter { c => c.name != KafkaConfig.EnableMetadataQuorumProp }.foreach(c => {
```

Comment:
@abbccdda `enable.metadata.quorum` has no documentation, so I had to exclude that explicitly from the test `KafkaApisTest.testDescribeConfigsWithDocumentationKip500` — is this something that needs to be fixed in AK?